src/components/SignUp.vue

```diff
@@ -69,7 +69,7 @@
           /></span>
         </button>
         <br />
-        <button class="btn btn-pill github mb-3 pad-google" type="button">
+        <button class="btn btn-pill github mb-3 pad-google g-signin2" type="button">
```

Comment:
is this class required ? For what purpose ?

Reply:
no